packages/opentelemetry-instrumentation-langchain/tests/test_tool_call_content.py

```
@@ -0,0 +1,161 @@
+"""
+Test for the fix of the issue where assistant message content is missing
+when tool calls are present in LangGraph/LangChain instrumentation.
+
+This test reproduces the issue reported in GitHub where gen_ai.prompt.X.content
+attributes were missing for assistant messages that contained tool_calls.
+"""
+
+import pytest
```

Ruff (0.12.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)